Domains/2-DisciplinePhysical/Civil/RoadRailAlignment/RoadRailAlignment.ecschema.xml

```diff
@@ -195,4 +195,23 @@
             <Class class="bis:GraphicalElement2d"/>
         </Target>
     </ECRelationshipClass>
+    <ECEntityClass typeName="ProfileDrawing" displayLabel="Profile Drawing" modifier="None" description="A bis:Drawing that showcases the variations in elevation of elements of interests along an lr:ILinearElement in a graphical representation.">
+        <BaseClass>bis:Drawing</BaseClass>
+        <ECProperty propertyName="VerticalExaggeration" typeName="double" />
+        <ECNavigationProperty propertyName="ProfileLocation" relationshipName="ProfileDrawingRefersToLocation" direction="Forward" displayLabel="Profile Location"/>
+    </ECEntityClass>
+    <ECRelationshipClass typeName="ProfileDrawingRefersToLocation" strength="referencing" modifier="None" description="Relationship associating a ProfileDrawing to a  ProfileDrawingLocation.">
+        <Source multiplicity="(0..*)" roleLabel="refers to main" polymorphic="false">
+            <Class class="ProfileDrawing"/>
+        </Source>
+        <Target multiplicity="(1..1)" roleLabel="is referenced by" polymorphic="false">
+            <Class class="ProfileDrawingLocation"/>
+        </Target>
+    </ECRelationshipClass>
+    <ECEntityClass typeName="ProfileDrawingLocation" displayLabel="Profile Drawing Location" modifier="None" description="The location, in terms of a range over a lr:ILinearElement and an elevation range, that a referencing ProfileDrawing captures.">
+        <BaseClass>bis:InformationRecordElement</BaseClass>
```

Comment:
I think the `BaseClass` should be [LinearLocationRecordElement](https://github.com/iTwin/bis-schemas/blob/14b8b72fc0766581add2978f3b7e4a566985fd19/Domains/1-Common/LinearReferencing/LinearReferencing.ecschema.xml#L123) according to its description.